Foundry textures rework and other stuff changed

DevDyna opened the pull request:
# Foundry Texture rework
- changed some textures to match more other dyed color blocks (like wool)
  - entirely changed textures :
    - Black
    - Brown
    - Blue
    - Magenta
  - partially changed textures : 
    - Light Gray
    - Red
    - Orange
    - Yellow
    - Lime
    - Light Blue
    - Pink
    - Cyan
  - unvaried textures :
  - Gray
  - Green
  - Purple
  - White

## Before
![2025-03-21_20 34 17](https://github.com/user-attachments/assets/dec6b7f1-1976-4105-a3bf-b7904c08bdbf)

## Later
![2025-03-21_21 40 37](https://github.com/user-attachments/assets/aadb9b75-82ed-4b25-8f5b-5aa44907a202)

### it include all :
- FireBricks
- Controllers (FireBricks parenting)
- Drains (FireBricks parenting)
- Tanks
- Windows

# Gui tweaks
- Added a fire image to indicate that slot was to "smelt stuff"
- Added small lines to on fluid tanks

![image](https://github.com/user-attachments/assets/4c4b6c9f-5efa-406b-82b7-9d488a09d93e)

# Other
- Updated jei categories images to match new textures
- Updated Fusion integration to match new textures
- Changed SoundType at Foundry controller , drain , tank and bricks to "sound more like bricks" intend of stone

DevDyna committed:
textures update

DevDyna committed:
Update foundry_controller.png

added a little flame on slot to indicate "items to smelt"
added small details on tank displayer

DevDyna committed:
updated all jei categories images

DevDyna committed:
Update .gitignore

DevDyna committed:
changed soundtype to nether bricks

DevDyna committed:
updated orange to be more saturated

JaisDK commented:
why didn't you come to the discord to discuss this?

DevDyna commented:
> why didn't you come to the discord to discuss this?

i didn't expected that there was "problems" about that..

JaisDK commented:
You’re not fixing a bug here, you’re changing the look of the mod, of course there should have been some communication about it.

JaisDK commented:
I will need a few changes to the textures before merging them. I am not aiming to have the exact same color palette as their wool or concrete counterparts.

The red and orange needs to be darker and more burnt, these are burnt bricks after all. The shading on red and orange is good they just need to be darker. Yellow is too vibrant. Lime I don't like, it looks like bamboo, the old one is better but could maybe do with some more detailing.

The fire icon on the GUI slots don't help alleviate any confusion, they just make it look like you need to put something special there and not just items, please remove those.

DevDyna committed:
reverted fire on gui slot to empty

DevDyna commented:
something like that could be good? 
(i ask before i start adding Fusion compat)

> red and orange

increased the contrast of the shading and darker

![image](https://github.com/user-attachments/assets/c81e3045-4a28-47bf-8199-0ba86757a039)
![image](https://github.com/user-attachments/assets/ef393000-4498-4984-a8fe-6329753d2f5b)

> yellow

reduced the brightness and removed some colors too bright

![image](https://github.com/user-attachments/assets/a7d9c6b6-654f-43c4-ac85-79c56f86fd19)

 
> lime

added a border and added some details

![image](https://github.com/user-attachments/assets/6a060b01-e305-49f5-a244-e7effb1bcdd5)

DevDyna committed:
red and orange more dark

DevDyna committed:
yellow less blight

DevDyna committed:
lime less "bamboo like"

JaisDK commented:
if the red could be a little more bordeaux and less blood I think we're there

DevDyna commented:
> if the red could be a little more bordeaux and less blood I think we're there

like that?

![image](https://github.com/user-attachments/assets/e19658ba-36fc-4d42-96bd-cd1d51a61595)

DevDyna committed:
Update red_fire_bricks.png

DevDyna committed:
updated lime stuff

DevDyna committed:
updated orange stuff

DevDyna committed:
updated yellow stuff

DevDyna committed:
updated all fusion stuff

JaisDK commented:
yes, is it ready to merge?

DevDyna commented:
> yes, is it ready to merge?

i need to update all red stuff and on later it's ready

JaisDK commented:
I'll set it to draft and you can change it back when it's ready

DevDyna committed:
updated red stuff

DevDyna committed:
updated red fusion stuff
